### DISCUSSION TITLE
BR-BT-01311-0152 message is wrong

### DISCUSSION THREAD

**Discussion by AnnaKSZ** [OP-TED/eForms-SDK#639] [Q&A] [answered]
Message code: "BR-BT-01311-0152"

SDK 1.7.0 message:
Deadline Receipt Requests (BT-1311) value must be later than Dispatch Invitation Tender (BT-130) value.

There were a fix:
https://github.com/OP-TED/eForms-SDK/commit/271624b35f492775456ba71a0a27e46381a70dba

SDK 1.7.1, 1.8.0 message:
Deadline for receipt of requests to participate' (BT-1311) must be later than 'Estimated date of dispatch of the invitations to submit tenders' (BT-130)

SDK 1.7.1. Fields.json 
<code>"id" : "BT-1311(d)-Lot",
...
"condition" : "{ND-LotTenderingProcess} ${(BT-1311(d)-Lot is present) and (BT-130-Lot is present)}",
"value" : "{ND-LotTenderingProcess} ${(BT-1311(d)-Lot < BT-130-Lot)}",
"severity" : "ERROR",
"message" : "rule|text|BR-BT-01311-0152"</code>

Please fix it from "later" to "earlier" and fix it in the other translation files as well.

Hungarian wrong:
"'A részvételi jelentkezések beérkezésének határideje' (BT-1311) később kell, hogy legyen, mint 'Az ajánlattételi felhívás közzétételének becsült dátuma' (BT-130)"

Hungarian good:
"'A részvételi jelentkezések beérkezésének határideje' (BT-1311) korábbi kell, hogy legyen, mint 'Az ajánlattételi felhívás közzétételének becsült dátuma' (BT-130)"

---

Other question about SDK 1.8.0. translation:
We have seen 1.8.0 and there are a lot of bad translation in the hungarian translation. 
Are these translation made by human translator? 
Are these the final translations? 
If not when we can get human translations?
If yes where can we ask to change these bad translations?
We see that we need to update the SDK regularly, so if we change a message we have to check and do each update process.

**Comment by bertrand-lorentz** [comment, marked as answer]
Thanks for your feedback.

We had indeed not updated the message for BR-BT-01311-0152 when we made the correction you link to.
We have already noticed this a few days ago, corrected the label in English, and removed all the translations to avoid any confusion.
This correction will be available in the next release of the SDK (1.9.0), and will then in the patches we will do for older version (1.8.1, 1.7.2, etc.)
The new message will be (re)translated in the other languages by our translators, but those translations might not be available yet for SDK 1.9.0.

For the general status of translation in hungarian : I know we have made some translations with the machine translation tool of the Commission, and then have them reviewed by translators. But I'm not sure what the current status is, I'll have to check with my colleagues.

**Comment by AnnaKSZ** [comment]
Thanks for your response
Anna
